### PR TITLE
Update README install docs

### PR DESCRIPTION
**Pull request by cherman2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,18 +10,21 @@
 
 Please note that this software is now an alpha release. Initial conda packages are available, but have not been tested in integration against our other plugins yet. For now, create a fresh conda environment for q2-fmt using the following command:
 
+OSX instructions
```
Comment (cherman2) [inline README.md]:
```suggestion
Mac OS instructions 
```

**Comment by colinvwood** [comment]
These are some pretty fundamental changes so let me mull this over for a while and I'll get back to you.

File: README.md [hunk 2]
```diff
@@ -10,18 +10,21 @@
...
+wget https://data.qiime2.org/distro/fmt/qiime2-fmt-2023.9-py38-linux-conda.yml
+conda env create -n qiime2-fmt-2023.9 --file qiime2-fmt-2023.9-py38-linux-conda.yml
 ```
 Then activate your new environment as usual.
 ```bash
-conda activate q2-fmt-alpha
+conda activate q2-fmt-2023.9
 ```
-Please be aware that `q2-fmt` will be the only visible plugin in this environment.
+Please be aware that `q2-fmt`  and  `q2-stats` will be the only visible plugins in this environment.
```
Comment (colinvwood) [inline README.md]: When I installed I also got vizard.

Comment (cherman2) [inline README.md]:
```suggestion
Please be aware that `q2-fmt`,  `q2-stats` and `q2-vizard`  will be the only visible plugins in this environment.
```

File: README.md [hunk 3]
```diff
@@ -10,18 +10,21 @@
...
 
+Mac OS instructions 
 ```bash
- conda create -n q2-fmt-alpha \
-   -c conda-forge \
-   -c bioconda \
-   -c https://packages.qiime2.org/qiime2/2023.2/tested/ \
-   q2-fmt q2cli
+wget https://data.qiime2.org/distro/fmt/qiime2-fmt-2023.9-py38-osx-conda.yml
+conda env create -n qiime2-fmt-2023.9 --file qiime2-fmt-2023.9-py38-osx-conda.yml
```
Comment (cherman2) [inline README.md]:
```suggestion
CONDA_SUBDIR=osx-64 conda env create -n qiime2-fmt-2023.9 --file qiime2-fmt-2023.9-py38-osx-conda.yml
conda activate qiime2-fmt-2023.9
conda config --env --set subdir osx-64
```

**Comment by cherman2** [comment]
I pulled this down and you are right the mac instructions for M1 works for my computer as well. @colinvwood

**Pull request merged**